great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -94,6 +97,7 @@ def get_line_chart(
         metric_type: alt.StandardType,
         domain_name: str,
         domain_type: alt.StandardType,
+        subtitle: Optional[str] = None,
```

Comment:
Necessary modification to make `get_line_chart` generic and work with both domains. I'm wary of bloating function signatures like this but this one simple change let's us reuse all the logic (since the only difference between the domains was a subtitle to reflect the column name).